test/test_methods.py

```diff
@@ -149,6 +149,29 @@ def test_load_coordinates(self):
                 for coords, expected in zip(coordinates, objects):
                     self.assertEqual(coords, expected)
 
+    def test_load_coordinates_scientific_notation(self):
+
+        # For some datasets that generate coords so close to zero that they end up in scientific notation
+
+        data = '7.9720694373e-05 44.6352243008'
+        with methods.tempinput(data) as path:
+            coordinates = methods.load_coordinates(path)
+            coords_list = list(coordinates)
+            self.assertEqual(len(coords_list), 1)
+            self.assertEqual(coords_list[0], (7.9720694373e-05, 44.6352243008, None, None))
```

Comment:
I realize now that we're comparing floating-point numbers, so `assertEqual()` isn't the best practice and may break at some point. Instead, what if we do:

```
ra, dec, pm_ra, pm_dec = coords_list[0]
self.assertAlmostEqual(ra, 7.9720694373e-05)
self.assertAlmostEqual(dec, 44.6352243008)
...
```

Same for the tests below (and feel free to fix the existing one above).

Reply:
good point I'll change that

Reply:
Done